source/whats-new.txt

```diff
@@ -41,6 +41,10 @@ What's New in 5.0
 
 New features of the 5.0 {+driver-short+} release include:
 
+- By default, the driver no longer checks types referenced in dot notation
+  unless the ``StrictFilter`` feature is explicitly
+  implemented. To learn more about this change, see the :ref:`Typescript fundamentals page <node-ts-type-safety>`.
```

Comment:
```suggestion
- By default, the driver no longer checks types referenced in dot notation
  unless the ``StrictFilter`` type annotation is explicitly
  used. To learn more about this change, see the :ref:`Typescript fundamentals page <node-ts-type-safety>`.
```

Reply:
Additionally, in our migration guide we made a **bold header call out** that this is a TypeScript only difference that does not effect runtime database queries/operations. I'm sure you have your own style rules but we felt it's important that there isn't confusion that our driver does special handling of keys with dots in them at runtime.